en/install.rst

```diff
@@ -46,8 +50,12 @@ developing with Zephir.
 
 Installing Zephir
 -----------------
+First, make sure that Zephir parser extension is installed and activated following tutorial_
+
 The Zephir compiler currently must be cloned from Github:
 
```

Comment:
Links at the end of the document, rather than sprinkled throughout.

Reply:
Sure